packages/blade/src/components/Box/LayoutPrimitivesDocs.tsx

```diff
@@ -0,0 +1,462 @@
+                        marginX="spacing.5" // adds horizontal margin
+                      >
+                        <Text>More Text</Text>
+                      </Box>
+                    */}
+                  </>
+                )
+              }
+  
+              export default App;
+              `}
+          </Sandbox>
+          <Text>
+            Blade supports multiple props like <Code>padding</Code>, <Code>paddingX</Code>,{' '}
+            <Code>paddingY</Code>, <Code>paddingTop</Code>, <Code>paddingRight</Code>,{' '}
+            <Code>paddingBottom</Code>, <Code>paddingLeft</Code> + similar props with{' '}
+            <Code>margin</Code>. And these props can have values such as <Code>spacing.3</Code> (Our
+            tokens), <Code>132px</Code> (absolute values), <Code>auto</Code>
+          </Text>
+        </Section>
+
+        <Section id="responsive-props">
+          <Heading size="large" marginBottom="spacing.3">
+            Responsive Props 📱 🖥
+          </Heading>
+          <Text marginBottom="spacing.4">
+            Check out how the code renders differently on screens when{' '}
+            <Code>{`{base: 'column', m: 'row'}`}</Code> is used.
+          </Text>
+          <SandboxProvider
+            code={`
+             import { Box, Text } from '@razorpay/blade/components';
+
+             function App(): JSX.Element {
+               return (
+                <>
+                  <Box display={{ base: 'none', m: 'block' }}><Text>Desktop View</Text></Box>
+                  <Box display={{ base: 'block', m: 'none' }}><Text>Mobile View</Text></Box>
+                  <Box 
+                    padding="spacing.4"
+                    marginTop="32px"
+                    display="flex"
+                    // Magic line of code 👇🏼
+                    flexDirection={{ base: 'column', m: 'row' }}
+                  >
+                    <Box
+                      flex="1"
+                      backgroundColor="surface.background.level2.highContrast"
+                      minHeight="spacing.10" 
+                    />
+                    <Box 
+                      flex="1" 
+                      backgroundColor="surface.background.level3.highContrast" 
+                      minHeight="spacing.10" 
+                    />
+                  </Box>
+                </>
+               )
+             }
+ 
+             export default App;
+            `}
+          >
+            <SandpackLayout>
+              <SandpackCodeEditor />
+              <SandpackPreview />
+              <Box display={{ base: 'none', m: 'block' }} width="100%">
+                <SandpackPreview style={{ width: '100%' }} />
+              </Box>
+            </SandpackLayout>
+          </SandboxProvider>
+          <Text marginTop="spacing.4">
+            All the props of Box component support responsive objects 🕺🏻. For which breakpoint to
```

Comment:
Lets add this before the demo, after the demo it seems kinda hidden

Reply:
Its not that important note so should be fine no? there is already some text before example so don't want to add too much there.